### PR TITLE
refactor: use IPC helpers in window-setup

### PR DESCRIPTION
**Pull request by miniak**
#### Description of Change
Use `ipcRendererUtils.invoke(Sync)` / `ipcMainUtils.handle` to eliminate duplicate code and properly propagate errors to the caller side.

We need to handle `loadURL` similar to #18990

#### Checklist
- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)

#### Release Notes
Notes: no-notes

### CODE REVIEW

File: lib/browser/guest-window-manager.js [hunk 1]
```diff
@@ -267,53 +271,42 @@ ipcMainInternal.on('ELECTRON_GUEST_WINDOW_MANAGER_INTERNAL_WINDOW_OPEN', functio
...
+    console.error(`Blocked ${event.sender.getURL()} from calling ${method}`)
+    throw new Error(`Access denied to guestId: ${guestContents.id}`)
   }
 
-  if (!canAccessWindow(event.sender, guestContents) || !windowMethods.has(method)) {
-    console.error(`Blocked ${event.sender.getURL()} from calling ${method} on its opener.`)
-    event.returnValue = null
-    return
+  if (!windowMethods.has(method)) {
+    console.error(`Blocked ${event.sender.getURL()} from calling ${method}`)
```
Comment (alexeykuzmin) [inline lib/browser/guest-window-manager.js]: Should this be different from the message up there?

File: spec/chromium-spec.js [hunk 2]
```diff
@@ -666,10 +666,10 @@ describe('chromium feature', () => {
       if (webview != null) webview.remove()
     })
 
-    it('does nothing when origin of webview src URL does not match opener', (done) => {
+    it('fails when origin of webview src URL does not match opener', (done) => {
       webview = new WebView()
       webview.addEventListener('console-message', (e) => {
-        expect(e.message).to.equal('')
+        expect(e.message).to.equal('null')
```
Comment (alexeykuzmin) [inline spec/chromium-spec.js]: The `'null'` string looks weird.

Comment (miniak) [reply spec/chromium-spec.js]: it does, but that's how `console.log(null)` ends up in `'console-message'`

File: lib/browser/guest-window-manager.js (hunk 1 above)
Comment (miniak) [reply lib/browser/guest-window-manager.js]: what do you mean exactly?

Comment (alexeykuzmin) [reply lib/browser/guest-window-manager.js]: I mean that you throw exceptions with two different errors, but log two exactly the same messages to the console.
Maybe the message should be different as well?

**Pull request merged**